src/cljs/athens/views/athena.cljs

```diff
@@ -231,7 +239,7 @@
                      :prefix   "Create page"
                      :preview  nil
                      :type     :page
-                     :query    query
+                     :query    nil
```

Comment:
Because this `result-el` is to show the `Create page: <page name>` result, and shouldn't highlight `page name`. I changed `highlight-match` to not try to do any highlighting if there's no query.